opencensus/proto/stats/metrics/metrics.proto

```diff
@@ -69,15 +70,19 @@ message MetricDescriptor {
   Type type = 4;
 
   // TODO: Consider to add LabelInfo{key, description} if needed.
+  // The label keys associated with the metric descriptor. Label keys must be
+  // unique and in sorted order.
```

Comment:
As discussed offline, we don't want to do too much validation. I think the order of keys should be left unspecified.

Reply:
That's right, removed. I think we don't want to validate if the tag keys are unique either.